libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2021-2022 GoodData Corporation
+
+export interface IOrganizationWhiteLabeling {
+    enabled: boolean;
+    faviconUrl?: string;
+    logoUrl?: string;
+    appleTouchIconUrl?: string;
+}
+
+export interface IOrganizationSettingsService {
+    /**
+     * Set whiteLabeling in organization.
+     *
+     * @returns promise
+     */
+    setWhiteLabeling(whiteLabeling: IOrganizationWhiteLabeling): Promise<void>;
+
+    /**
+     * Request whiteLabeling from organization.
+     *
+     * @returns promise of theme object reference
+     */
+    getWhiteLabeling(): Promise<IOrganizationWhiteLabeling | undefined>;
```

Comment:
This method is not necessary. It can be left here but we will not use it. Obtaining the settings from organization requires permission to read organization data, i.e., you would have to be domain admin which most users that will use the UI will not be and they will end up with exception when UI app bootstraps. This is the reason why this method was not mentioned in the SDK change doc proposal. 

There are two possibilities: either delete it (I would prefer that as no one will use it) or mention this permissions requirement in the JSDoc.

Reply:
Thank you, I will rather delete this method.